tests/lightning_qubit/test_measurements_class.py

```diff
@@ -613,6 +610,10 @@ def test_double_return_value(self, shots, measurement, obs0_, obs1_, lightning_s
             # allclose -> absolute(r - e) <= (atol + rtol * absolute(e))
             assert np.allclose(r, e, atol=dtol, rtol=dtol)
 
+    @pytest.mark.skipif(
+        device_name == "lightning.gpu",
+        reason="lightning.gpu does not support out of order prob.",
+    )
```

Comment:
@multiphaseCFD Is there any 3rd-party blocker to add the out-of-order prob support?

Reply:
In this case, I'm wondering if we could: order the wires, calculate the probs, and transpose back to the unordered set of wires. @multiphaseCFD, did we try it already? Do you see any blockers for it? I'm just thinking about it, nothing to do now, of course.